### ISSUE TITLE
Diagram in the Memory Map is Incorrect

### ISSUE THREAD

**Issue by jtgans** [c128lib/Reference#43]
During reset, the 128 defaults the RCR to $04, which puts the 1K common area of RAM at the bottom of the address space, overlapping the zero page, the stack page, and $02 and $03. The diagram currently shows it at the top of the address space, which is incorrect.

**Comment by intoinside** [comment]
Yes, my fault. Thank you for pointing it!

**Comment by johnpalermo** [comment]
It was my diagram and maybe I labeled it incorrectly. I refer to $FC00 - $FEFF as common RAM. I cannot find any references of how this area is mapped, so the assumption is that it can be used as free RAM. However, I'm not sure if this area is common to RAM 0 and RAM 1, or is distinct in each bank. Maybe rewording to free RAM would be better, unless there is some other mapping or purpose for this area. Here, I'm not referring to the common RAM at the bottom of memory, which I agree is Page 0 and System RAM at $0000 - $03FF.

**Comment by jtgans** [comment]
By default it's mapped as ROM normally, or just bank 0 or 1 RAM, no common area by default. To make it common, bits 4 and 5 in thr MMUCR have to be set and the MMURCR has to have the bit set for it.

Basically, it's good if you're executing out of bank 1 and need IRQ vectors to still be valid.

**Comment by intoinside** [comment]
As @jtgans says, the common area is controlled by register $d506. Bits 0-1 defines the size while bits 2-3 defines the position (no common area, bottom, top, or both). On reset, the register is set to $04 and activates 1KB of common ram, bottom only.
This is the statup sequence
![image](https://github.com/c128lib/Reference/assets/8144124/79d8b48f-888f-4750-afe0-47acd96063eb)

These are the startup bytes at $e04b
![image](https://github.com/c128lib/Reference/assets/8144124/c9074176-b93a-4315-bf0b-fee1212f1845)

Regarding the diagram, probably the simplest thing is to eliminate the references to the common memory that appears at $FC00?
Would another diagram be helpful to show different combinations of common ram?

**Comment by johnpalermo** [comment]
I'll correct the diagram and resubmit it. I'll make $FC00 to $FEFF part of the Kernal ROM as referenced in Mapping The Commodore 128. 

Another diagram to show different combinations of common ram is a good idea.

**Comment by johnpalermo** [comment]
I've updated my ods file and attached it. Please feel free to use it in the reference repository, correct it, change it, including the colouring and formatting. It took me a bit of time to originally create it, so thought it could save another person some time. In this revision, I removed the reference to $FC00 - $FEFF and included it into the block of Kernal ROM.

[Commodore 128 Memory Map.ods](https://github.com/user-attachments/files/15753220/Commodore.128.Memory.Map.ods)

**Comment by intoinside** [comment]
Updated
